docs/core/development/policy/release-process.rst

```diff
@@ -184,31 +184,12 @@ Prepare the branch
 #. Commit the version and ``README.rst`` changes.
 #. Submit the ticket for review
 #. Pause until the ticket is reviewed and accepted.
-#. Tag the release.
+#. Tag the release. Github Actions will upload the dist to PyPI
```

Comment:
Maybe instead of `Tag the release' we should suggest using the GitHub Release UI and add the changelog there
and create a tag via the GItHub UI at https://github.com/twisted/twisted/releases/new
This is to replace the removed  ' Cut the tarballs & installers '

And as a separate paragraph inform that Twisted package is distribute only via PyPI.... to replace the removed `Distribute` section.

The `How to do a release candidate` section should also be updated.


-------
```suggestion
#. Use the `GitHub Create Release UI <https://github.com/twisted/twisted/releases/new>`_ the make a new release and tag.
#. Create a tag using the format `twisted-VERSION` based on the latest commt on the release branch that was approved after the review.
#. Use `Twisted VERSION` as the name of the release.
#. Add the release notes to GitHub Release page.
#. Make sure 'This is a pre-release` is not checked.
#. Github Actions will upload the dist to PyPI when a new tag is pushed to the repo. PyPI is the only canonical source for Twisted packages.
```